package.json

```diff
@@ -21,6 +21,6 @@
   },
   "dependencies": {
     "eu.netherlands3d.sun": "1.0.0",
-    "eu.netherlands3d.snapshots": "1.0.0"
+    "eu.netherlands3d.snapshots": "1.0.1"
```

Comment:
Ik mis de filebrowser package als dependency hier; kan dat kloppen?

Reply:
zit er in nu